model/metrics/database-metrics.yaml

```
@@ -108,3 +108,16 @@ groups:
     attributes:
       - ref: db.client.connections.pool.name
         requirement_level: required
+
+  - id: metric.db.client.connections.usage
+    type: metric
+    metric_name: db.client.connections.usage
+    stability: experimental
```

is this supported by the yaml?

```suggestion
    deprecated: "Replaced by `db.client.connections.count`."
```